File: app/src/main/java/com/bing/mvvmbase/MyAppliction.java
package com.bing.mvvmbase;

import com.bing.mvvmbase.base.BaseApplication;
import com.squareup.leakcanary.LeakCanary;

public class MyAppliction extends BaseApplication {
	@Override
	public void onCreate() {

		if (LeakCanary.isInAnalyzerProcess(this)) {
			return;
		}
		LeakCanary.install(this);
		super.onCreate();
	}
}

File: app/src/main/java/com/bing/mvvmbase/MyAppliction.kt
package com.bing.mvvmbase

import com.bing.mvvmbase.base.BaseApplication
import com.squareup.leakcanary.LeakCanary

class MyAppliction : BaseApplication() {
        override fun onCreate() {

                if (LeakCanary.isInAnalyzerProcess(this)) {
                        return
                }
                LeakCanary.install(this)
                super.onCreate()
        }
}
